tensorboard/uploader/flags_parser.py

```diff
@@ -177,6 +177,11 @@ def define_flags(parser):
         "list", help="list previously uploaded experiments"
     )
     list_parser.set_defaults(**{SUBCOMMAND_FLAG: SUBCOMMAND_KEY_LIST})
+    list_parser.add_argument(
+        "--json",
```

Comment:
@bileschi  As pointed out by @wchargin, "prior art" varies. So I opted to err on the side of conciseness. Also see my earlier comments regarding this.